www/src/components/ImportApi.js

```diff
@@ -0,0 +1,52 @@
+import { faCopy } from '@fortawesome/free-solid-svg-icons/faCopy';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import styled from 'astroturf';
+import copy from 'copy-text-to-clipboard';
+import React from 'react';
+import OverlayTrigger from 'react-bootstrap/OverlayTrigger';
+import Tooltip from 'react-bootstrap/Tooltip';
+
+const Link = styled('span')`
+  font-size: 1rem;
+  padding: 0 0.5rem;
+  cursor: pointer;
+`;
+
+const Keyword = styled('span')`
+  color: #a626a4;
+`;
+
+const Code = styled('code')`
+  padding: 0;
+  display: inline-block;
+  color: #50a14f;
+  background-color: transparent;
+  margin-bottom: 1rem;
+`;
+
+const CopyImport = ({ name }) => {
+  const textToCopy = `import ${name} from 'react-bootstrap/${name}'`;
+
+  return (
+    <OverlayTrigger
+      overlay={
+        <Tooltip id={`copy-${name}-import-tooltip`}>Copy import code</Tooltip>
```

Comment:
Could we change the overlay text to indicate to the end-user that it's been copied after they click on it? E.g. changing the text from `Copy import code` to `Copied!`.

Reply:
Done :) Thanks for the review. The text is now changed to `Copied!` when the button is clicked and restored back to the original text after 2 seconds.

Reply:
@bpas247 can you handle merging this then?